### PR TITLE
Tiny Getting Started documentation fix: port-forward service instead of pod

### PR DESCRIPTION
**Pull request by toonsevrin**
I was not sure why you guys were making this more difficult then it is :laughing:

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ You can now access the web interface by port forwarding the UI pod (mind the
 label selector) and enter `localhost:8081` in your browser:
 
 ```bash
-kubectl port-forward "$(kubectl get pod -l name=postgres-operator-ui --output='name')" 8081
+kubectl port-forward svc/postgres-operator-ui 8081:80
```
Comment (FxKu) [inline docs/quickstart.md:163]:
```suggestion
kubectl port-forward svc/postgres-operator-ui 8081:8081
```

with this it works for me

Comment (toonsevrin) [reply docs/quickstart.md]: My apologies :sweat_smile:, I've updated the PR.

**Comment by FxKu** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

**Comment by FxKu** [comment]
Thanks @toonsevrin for your contribution!

**Pull request merged**